packages/e2e-test-utils/src/customFonts.js

```diff
@@ -13,11 +13,54 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+function parseText(optionsText) {
+  return optionsText.map((option) => {
+    const font = option.split('\n');
+    return { name: font[0], url: font[1] };
+  });
+}
+
+/**
+ * Get custom font list
+ *
+ * @return {Promise<[]>}
```

Comment:
To address lint warnings:

```suggestion
 * @return {Promise<[Object<{name: string, url: string}]>} Fonts list.
```